bootstrap/api/endpoint_test.go

```diff
@@ -160,7 +160,7 @@ func dec(in []byte) ([]byte, error) {
 func newService(auth mainflux.AuthServiceClient, url string) bootstrap.Service {
 	things := mocks.NewConfigsRepository()
 	config := mfsdk.Config{
-		BootstrapURL: url,
+		ThingsURL: url,
```

Comment:
Why `ThingsURL`? It looks that caller is calling this `newService()` funtion with the URL of Bootstrap server, right? And we are giving this to ThingsURL.

Reply:
`ThingsURL` is used to reach `things` service API endpoints.
Bootstrap test server is created after `newService` with
`bs := newBootstrapServer(svc)`
and then you can test bootstrap endpoints using bs.URL
```
	for _, tc := range cases {
		req := testRequest{
			client:      bs.Client(),
			method:      http.MethodPost,
			url:         fmt.Sprintf("%s/configs", bs.URL),```

Reply:
Naming of these functions is confusing. Calling something called `newService()` inside of Boostrap code, I would expect that it gives me back Boostrap service.

Also, take a look at the callers - what URLs do they provide? URLs of Things? How do you know where Things is - this is something that user should tell you via ENV var, no?